srcpkgs/singularity/INSTALL

```diff
@@ -0,0 +1,6 @@
+case "${ACTION}" in
+post)
```

Comment:
```suggestion
post)
if [ "$(readlink -f . 2>/dev/null || echo .)" != "/" ]; then
	exit 1
else
	mkdir -p var/lib/singularity/mnt/session
	chown root:_singularity usr/libexec/singularity/bin/starter-suid
	chmod 4755 usr/libexec/singularity/bin/starter-suid
fi
```
Maybe do smth like this to prevent changing the binary to a potentially wrong group when executing xbps-install with `--rootidr`

Reply:
I like the idea, but I'd prefer to have it as a general helper, otherwise this will have to be repeated across a few INSTALL scripts (including XBPS; it's not a huge issue there because the gid for `xbuilder` is constant, but still not great).

The helper shouldn't be added here though, so I will do it afterwards.

Reply:
I added the check and replaced the `mkdir` with `make_dirs` in the template.